extensions/amp-access/0.1/amp-access-vendor.js

```diff
@@ -41,6 +41,12 @@ export class AccessVendorAdapter {
     this.vendorName_ = user().assert(configJson['vendor'],
         '"vendor" name must be specified');
 
+    /** @const @private {JSONType} */
+    this.vendorConfig_ = configJson[this.vendorName_];
+
+    /** @const @private {boolean} */
+    this.isPingbackEnabled_ = !configJson['noPingback'];
```

Comment:
Asked a question about it above. If you don't need pingback at this time (though I think you probably do need it) you can just do a no-op in your service.